cli/tools/lint.rs

```diff
@@ -182,15 +172,18 @@ pub fn print_rules_list(json: bool) {
     // The rules should still be printed even if `--quiet` option is enabled,
     // so use `println!` here instead of `info!`.
     println!("Available rules:");
-    for rule in lint_rules {
+    for rule in lint_rules.into_iter() {
```

Comment:
I think if we use `iter()` instead of `into_iter()` here, we won't need to `unwrap` on the line 164.

Reply:
Thanks, had to do some more changes, but removed unwrap.